api/controllers/services/ckan/agencyService.js

```diff
@@ -7,11 +7,11 @@ class AgencyService extends AbstractService {
         this.agencyServiceResourceId = this.config.agency_service_resource;
         this.serviceTaxonomyResourceId = this.config.service_taxonomy_resource;
         this.agencyResourceId = this.config.agency_resource;
-        this.uri = `${this.host}/api/3/action/datastore_search_sql?sql=SELECT * FROM "${this.agencyResourceId}" ${this.tableName} `;
+        this.uri = `${this.host}/api/v1/organization/handson_central_ohio/dataset/${this.agencyResourceId}/query?_format=json`;
         this.joinQuery = `
-        INNER JOIN "${this.agencyServiceResourceId}" agency_service ON agency."AGENCY_ID" = agency_service."AGENCY_ID" 
-        INNER JOIN "${this.serviceTaxonomyResourceId}" service_taxonomy ON agency_service."AGENCY_ID" = service_taxonomy."AGENCY_ID" 
-        AND agency_service."LINE_NUMBER" = service_taxonomy."LINE_NUMBER"
+        &where= INNER JOIN "${this.agencyServiceResourceId}" agency_service ON agency."agency_id" = agency_service."agency_id"
```

Comment:
can we do JOINs in the new API?

Reply:
no this part of code is not used and will be replaced in my updated PR.